src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -2548,7 +2548,12 @@ private static unsafe void FreeArray(byte** arr, int length)
 
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of 'uufvhtmm.4sl'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 208
208:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```